pkgs/data/fonts/julia-mono/default.nix

```diff
@@ -14,9 +15,12 @@ in fetchzip {
 
   meta = {
     description = "A monospaced font for scientific and technical computing";
+    longDescription = ''
+      JuliaMono is a monospaced typeface designed for use in text editing environments that require a wide range of specialist and technical Unicode characters. It was intended as a fun experiment to be presented at the 2020 JuliaCon conference in Lisbon, Portugal (which of course didn’t physically happen in Lisbon, but online).
+    '';
     maintainers = with lib.maintainers; [ suhr ];
     platforms = with lib.platforms; all;
```

Comment:
with lib should be moved to meta.

Reply:
Oops. New pr?

Reply:
More like a little cleanup. Going to put this into my clean collection.